PennyMe/ViewController.swift

```diff
@@ -102,6 +107,10 @@ class ViewController: UIViewController, UITextFieldDelegate, UIGestureRecognizer
             forAnnotationViewWithReuseIdentifier:MKMapViewDefaultAnnotationViewReuseIdentifier
         )
 
+        // check settings toggles
+        updateIncludedStates()
```

Comment:
New method that updates a list that keeps track of which toggles at the settings screen are switched on - so which machines should be shown